src/components/Card.tsx

```diff
@@ -1,27 +1,36 @@
 import React, { ReactNode } from 'react';
+import parse from 'html-react-parser';  // Importing parse function directly
 
 interface CardProps {
   title: string;
+  details?: string;  // New details prop
   image?: string;
   url?: string;
   children?: ReactNode; 
 }
 
-export function Card({ title, image, url }: { title: string; image?: string, url?:string }) {
+export function Card({ title, details, image, url }: CardProps) {
     return (
       <a
         href={url}
-        className="relative bg-backgroundLight rounded-lg w-full h-36 md:h-52 lg:h-72 mb-5 overflow-hidden"
+        className="relative bg-backgroundLight rounded-lg w-full w-[380px] max-w-full h-17 md:h-17 lg:h-36 m-2 overflow-hidden flex items-center justify-center"
```

Comment:
Constant width (w-[380]) could cause problems with some devices.

Reply:
Got some of these there, will do them alternatively

Reply:
true, fix'd